test/e2e/network/ingress.go

```diff
@@ -622,6 +622,7 @@ var _ = SIGDescribe("Loadbalancing: L7", func() {
 			}
 			if jig.Ingress == nil {
 				ginkgo.By("No ingress created, no cleanup necessary")
+				return
```

Comment:
golint complains
```
Errors from golint:
test/e2e/network/ingress.go:626:11: if block ends with a return statement, so drop this else and outdent its block
```

Reply:
Yes, right.
It is necessary to block retesting until fixing the above.

/hold

Reply:
Fixed